google-cloud-tools-plugin/ultimate/build.gradle

```diff
@@ -37,7 +37,7 @@ dependencies {
     compile files(project(':google-cloud-tools-plugin').sourceSets.main.output)
     compile files("${System.properties['java.home']}/../lib/tools.jar")
     compile ('com.google.cloud.tools:appengine-plugins-core:' + toolsLibVersion) {
-        exclude group: 'com.google.guava', module: 'guava'
```

Comment:
I think that guava is being transitively passed to the main module from another dependency.
Since it's not being passed to the ultimate module, and build completes successfully without it, I thought of getting rid of it.
I added the common library dependency to the ultimate module, and at the time just copied the common library dependency bit without checking if this exclusion was necessary.